src/test/regress/spec/isolation_move_placement_vs_move_placement.spec

```diff
@@ -46,6 +67,11 @@ step "s2-commit"
 	COMMIT;
 }
 
+step "s2-wait-and-commit"
+{
+	COMMIT;
+}
```

Comment:
As discussed over chat this is now the same as the `s2-commit` step right above. So let's remove this one.